smartmin/csv_imports/models.py

```diff
@@ -19,6 +19,14 @@ def generate_file_path(instance, filename):
 
 
 class ImportTask(SmartModel):
+    PENDING = 'PENDING'
+    STARTED = 'STARTED'
+    RUNNING = 'RUNNING'
+    SUCCESS = 'SUCCESS'
+    FAILURE = 'FAILURE'
+
+    READY_STATES = [SUCCESS, FAILURE]
```

Comment:
Right I got those from Celery even if it is not guaranteed that the will always be in sync.
Also for task_status I wanted to keep the status method for backward compatibility in case it was called by someone